Community/ClickaTell/automation/lib/python/community/clickatell/__init__.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
`SyntaxError: Non-ASCII character in file '/opt/openhab2/conf/automation/lib/python/community/clickatell/__init__.py', but no encoding declared; see http://www.python.org/peps/pep-0263.html for details`

I'll put it back in my coming PR.